### PR TITLE
test: add new nrt on executions part

### PR DESCRIPTION
**Pull request by z25clave**
## Type of modification

* [ ] Breaking change
* [ ] New Feature
* [x] Bug Fix
* [ ] Chore (refactor, documentation, tests... all the changes with no impact on ARA functionalities.) 

## Changes description

**Add some quick description about what your PullRequest change in
ARA's feature/behavior**

## Technical description

**Add a quick description about what your PullRequest change in 
ARA's code. (example: add a processor check in...)**

## PR CheckList

Please make sure your PullRequest respect all those items :

* [x] Your PR's title has the prefix : `feat:`, `fix:` or `chore:`
* [ ] Unless your PR is related to a Hotfix (and approved by a ARA maintener), the targeted branch for it is the branch `version/X.Y.Z` and **not** `master`
* [ ] You have asked a review from one of the ARA maintainer in your PR.
* [ ] If your PR is related to an issue, add the issue's number in it.
* [ ] All the code you added is documented.
* [ ] All the code you added is tested and the  tests are in success.
* [x] You already signed the [Contributor License Agreement](https://github.com/Decathlon/ara/blob/master/contributor-licence-agreement.adoc) and give us the document

### CODE REVIEW

**Review by BenjaminDupont — APPROVED** [review]

File: client/test/cypress/fixtures/executions_history.json [hunk 1]
```diff
@@ -0,0 +1,327 @@
+{
+  "id": 4,
+  "branch": "develop",
+  "name": "day",
+  "release": "v3",
+  "version": "benjAndCaro",
```
Comment (omar-chahbouni-decathlon) [inline client/test/cypress/fixtures/executions_history.json:6]: please change the version value

File: client/test/cypress/fixtures/executions_history.json [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+      },
+      "type": {
+        "code": "api",
+        "name": "Integ. APIs",
+        "source": {
+          "code": "api",
+          "name": "API",
+          "letter": "A",
+          "technology": "POSTMAN",
+          "vcsUrl": "https://github.com/decathlon/ara/tree/{{branch}}/server/src/main/resources/demo/collections/",
```
Comment (omar-chahbouni-decathlon) [inline client/test/cypress/fixtures/executions_history.json:134]: please make it more generic, like for example: https://vcs-url.com/company/tree/{{branch}}/collections

File: client/test/cypress/fixtures/executions_history.json [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+      },
+      "type": {
+        "code": "desktop",
+        "name": "HMI Desktop",
+        "source": {
+          "code": "web",
+          "name": "Web",
+          "letter": "W",
+          "technology": "CUCUMBER",
+          "vcsUrl": "https://github.com/decathlon/ara/tree/{{branch}}/generated-cucumber-report/src/main/resources/ara/demo/features/",
```
Comment (omar-chahbouni-decathlon) [inline client/test/cypress/fixtures/executions_history.json:239]: idem here

**Review by omar-chahbouni-decathlon — CHANGES_REQUESTED** [review]
Please also update your commit message: feat -> test

**Comment by omar-chahbouni-decathlon** [comment]
please sign the licence/cla and fix the semantic pull request -> test: add new nrt on executions part

**Review by omar-chahbouni-decathlon — APPROVED** [review]

**Pull request merged**